tests/test_mission.py

```diff
@@ -699,3 +699,20 @@ def test_mission_neutral(self):
         self.assertTrue(sweden2 is not None)
         self.assertEqual(len(sweden2.ship_group), 1)
         self.assertEqual(len(sweden2.plane_group), 1)
+
+    def test_mission_save_pictureFileName(self):
+        m = dcs.mission.Mission(terrain=dcs.terrain.Caucasus())
+        image_path = 'tests/images/blue.png'
+        reskey_B = m.add_picture_blue(image_path)
+        reskey_R =m.add_picture_red(image_path)
```

Comment:
sorry this hit my eye :D

```suggestion
        reskey_R = m.add_picture_red(image_path)
```